app/lib/features/news/model/news_post_color_data.dart

```diff
@@ -1,12 +1,61 @@
-
 import 'package:flutter/material.dart';
 
-List<Color> newsPostColors = [
-  Colors.brown,
-  Colors.red,
-  Colors.orange,
-  Colors.deepPurple,
-  Colors.purple,
-  Colors.indigo,
-  Colors.pink,
+class PostColorScheme {
+  final Color backgroundColor;
+  final Color foregroundColor;
+  final Color linkColor;
+
+  const PostColorScheme({
+    required this.backgroundColor,
+    required this.foregroundColor,
+    required this.linkColor,
+  });
+}
+
+final List<PostColorScheme> postColorSchemes = [
+  PostColorScheme(
+    backgroundColor: Colors.pinkAccent, // Purple 200
+    foregroundColor: Colors.white, // Deep Purple 900
+    linkColor: Colors.lightBlueAccent.shade700, // Blue 800
+  ),
+  PostColorScheme(
+    backgroundColor: Colors.green, // Purple 200
+    foregroundColor: Colors.white, // Deep Purple 900
+    linkColor: Colors.lightBlueAccent.shade100, // Blue 800
```

Comment:
```suggestion
    linkColor: Colors.lightBlue.shade900
```

[Color Contrast Ratio of 2.7](https://colors.artyclick.com/contrast-color-finder/?color1=#4caf50&color2=#01579b)